src/workload.rs

```diff
@@ -531,6 +555,11 @@ impl WorkloadStore {
                     .insert((service_sock_addr, port.target_port as u16));
             }
         }
+        if let Some(tx) = self.cert_tx.as_mut() {
+            if tx.blocking_send(widentity).is_err() {
```

Comment:
blocking_send is only when not in async runtime. This panics. We should use try_send instead and ignore the error. This will fail if we have 100 identities queued up - no problem, this is prefetch anyways, not required just an optimization. Without try_send once we hit 100 identities we would block xds on CA - not good

Reply:
done